ansible_base/rbac/managed.py

```diff
@@ -151,6 +151,8 @@ class TeamMember(TeamMixin, ManagedActionBase):
     'action_base': ManagedActionBase,
 }
 
+MANAGED_ROLES = [role.name for role in managed_role_templates.values() if hasattr(role, "name")]
```

Comment:
This runs at import time, and there is a setting that allows changing the final value, which won't be ready until apps are ready. So you can't do it at import time. Here's the data structure I'd suggest from test_app:

```python
In [7]: [r.name for r in permission_registry._managed_roles.values()]
Out[7]: 
['Team Member',
 'Team Admin',
 'Organization Admin',
 'Organization Member',
 'Cow Admin',
 'Cow Mooer']
```

The Cow roles are intended.

Reply:
Made some changes to use the permission registry instead. Do the new changes address the issue properly?